algorithms/src/snark/varuna/varuna.rs

```diff
@@ -549,13 +543,13 @@ where
         }
 
         // Compute the AHP verifier's query set.
-        let (query_set, verifier_state) = AHPForR1CS::<_, MM>::verifier_query_set(verifier_state);
+        let query_set = prover_state.verifier_state.as_ref().unwrap().query_set();
```

Comment:
Ditto: we introduce a `ProverState::query_set()` method

```suggestion
        let query_set = prover_state.query_set();
```

Reply:
c894c4bc6